### PR TITLE
Add ronhedwig's review

### PR DESCRIPTION
**Pull request by ronhedwigzape**

### CODE REVIEW

**Comment by ronhedwigzape** [comment]
kadakul bot

**Comment by ronhedwigzape** [comment]
haha

File: src/plugins/vuex-store/modules/reviews/ron-hedwig.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+         *                     examples: (come on, examples? really?)
+         *                     this is REQUIRED (please be brutally honest)
+         *
+         */
+
+        // here you go:
+        review: {
+            as     : ['student', ''],
+            rating : 5,
+            content: 'I am a student of Sir Arvic since my Senior Highschool. When I was Grade 11, I am new to programming and Sir Arvic was our teacher. He taught programming to us so right and fluently. Even that has no idea of programming can learn from him. I promise he\'s better than watching Youtube. He\'s also better at motivating students that are so down in programming. Also optimistic and will everyday make you special. Because of Sir Arvic, I am now good at programming, coding, communicating, and collaborating with people. Lastly, I am thankful that I\'m one of his students. Sir Arvic is best of best! 😎'
```
Comment (kulotsystems) [inline src/plugins/vuex-store/modules/reviews/ron-hedwig.js]: I'm so flattered @ronhedwigzape. But can you make this a little bit shorter? 😸

**Review by kulotsystems — CHANGES_REQUESTED** [review]
I'm so flattered @ronhedwigzape. But can you make your review a little bit shorter? 😸

File: src/plugins/vuex-store/modules/reviews/ron-hedwig.js (hunk 1 above)
Comment (ronhedwigzape) [reply src/plugins/vuex-store/modules/reviews/ron-hedwig.js]: Ok sir @kulotsystems

**Review by kulotsystems — APPROVED** [review]
Thank you so much @ronhedwigzape!

**Comment by ronhedwigzape** [comment]
I have now an update on my review Sir @kulotsystems. Please approve, thanks.

**Pull request merged**